packages/neutron-sdk/src/bindings/msg.rs

```diff
@@ -126,6 +126,36 @@ pub enum NeutronMsg {
     /// SubmitAdminProposal sends a proposal to neutron's Admin module.
     /// This type of messages can be only executed by Neutron DAO.
     SubmitAdminProposal { admin_proposal: AdminProposal },
+
+    /// TokenFactory message.
+    /// Contracts can create denoms, namespaced under the contract's address.
+    /// A contract may create any number of independent sub-denoms.
+    CreateDenom { subdenom: String },
+    /// TokenFactory message.
+    /// Contracts can change the admin of a denom that they are the admin of.
+    ChangeAdmin {
+        denom: String,
+        new_admin_address: String,
+    },
+    /// TokenFactory message.
+    /// Contracts can mint native tokens for an existing factory denom
+    /// that they are the admin of.
+    MintTokens {
+        denom: String,
+        amount: Uint128,
+        mint_to_address: String,
+    },
+    /// TokenFactory message.
+    /// Contracts can burn native tokens for an existing factory denom
+    /// that they are the admin of.
+    /// Currently, the burn from address must be the admin contract.
+    BurnTokens {
+        denom: String,
+        amount: Uint128,
+        /// Must be set to `""` for now
+        burn_from_address: String,
```

Comment:
@albertandrejev why is it so? I have copy pasted this part from your code in neutron wasmbindings.

Reply:
It was copy pasted from osmosis wasmbinding as well. https://github.com/osmosis-labs/osmosis/blob/e871beddfd4b7abc473c6fd7b9f5cfafbe9769b8/wasmbinding/bindings/msg.go#L46